sections/escrow/components/StakingRewardsTab/StakingRewardsTab.tsx

```diff
@@ -112,7 +110,9 @@ const StakingRewardsTab: React.FC = () => {
 						txHash: transaction.hash,
 						onTxConfirmed: () => {
 							setTransactionState(Transaction.SUCCESS);
-							escrowDataQuery.refetch();
+							setTimeout(() => {
```

Comment:
maybe refactor to utils call it, delayedRefetch if this is a common pattern

Reply:
Yeah I was thinking about this. Let's do it later tho